test/src/main/java/org/apache/accumulo/test/functional/SlowIterator.java

```diff
@@ -50,21 +51,37 @@ public static void setSeekSleepTime(IteratorSetting is, long t) {
     is.addOption(SEEK_SLEEP_TIME, Long.toString(t));
   }
 
+  public static void sleepUninterruptibly(IteratorSetting is, boolean b) {
+    is.addOption(SLEEP_UNINTERRUPTIBLY, Boolean.toString(b));
+  }
+
+  private void sleep(long time) throws IOException {
```

Comment:
I checked these changes by jstacking a tablet server running the new test and verifying that compaction threads were sleeping both ways.